src/processors.ts

```diff
@@ -18,8 +21,12 @@ export function createProcessor(processPart: PartProcessor): TemplateTypeInit {
   }
 }
 
-export function processPropertyIdentity(part: TemplatePart, value: unknown): void {
-  part.value = value instanceof Node ? value : String(value)
+export function processPropertyIdentity(part: TemplatePart, value: unknown, state: unknown): void {
+  if (part instanceof InnerTemplatePart) {
+    part.template.content.replaceChildren(new TemplateInstance(part.template, state))
```

Comment:
This is the weight-bearing portion of this diff.

I don't like how much is going on in this line, I wish it were conceptually simpler.

Also, the current type signatures don't afford consumers to forward along their processor, so the default behavior wouldn't work for consumers that rely on `propertyIdentityOrBooleanAttribute` behavior instead of (the default) `propertyIdentity` behavior.

I'm exploring a more flexible change.